exercises/scrabble-score/pubspec.yaml

```diff
@@ -0,0 +1,3 @@
+name: 'scrabble_score'
+dev_dependencies:
+  test: '<0.13.0'
```

Comment:
In the create_exercise_test.dart, extend the timeout to 2 minutes. As the tests are timing out on Travis CI.

Though the builds are not failing properly, so I'm going to make another ticket for it. So I'll have to double check the contents to ensure the results are correct.

Reply:
I think you may have meant `exercises_test.dart`?

I changed the Duration to 120 seconds.

Reply:
I stand corrected, yep.